docs/framework/interop/exposing-com-components.md

```diff
@@ -22,7 +22,7 @@ ms.workload:
   - "dotnet"
 ---
 # Exposing COM Components to the .NET Framework
-This section summarizes the process needed to expose an existing COM component to managed code. For details about writing COM servers that tightly integrate with the .NET Framework, see [Design Considerations for Interoperation](http://msdn.microsoft.com/library/b59637f6-fe35-40d6-ae72-901e7a707689).  
+This section summarizes the process needed to expose an existing COM component to managed code. For details about writing COM servers that tightly integrate with the .NET Framework, see [Design Considerations for Interoperation](https://msdn.microsoft.com/library/b59637f6-fe35-40d6-ae72-901e7a707689(v=vs.100)) in the MSDN Library.
```

Comment:
let's not add MSDN library here since soon it'll be moving to docs archive. We actually have an issue to remove that sentence from docs.